code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -130,12 +130,18 @@
 	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY
 	taste_description = "smoke"
 
+/datum/reagent/smoke_powder_activator
+	name = "Smoke Activator"
+	description = "Activates smoke powder"
+	reagent_state = LIQUID
+	color = "#C8C8C8"
+
 /datum/reagent/sonic_powder
 	name = "Sonic Powder"
 	description = "Makes a deafening noise."
 	reagent_state = LIQUID
 	color = "#C8C8C8"
-	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY
+	chem_flags = CHEMICAL_RNG_GENERAL 
```

Comment:
? Merge conflict resolution issue?

Reply:
No, it was more that smoke powder activator would be totally garbage to get from botany or floorpill. I did that intentionally to make sure it didn't dilute either pool?

Reply:
wait. where the fuck is this?